app/views/organizations/index.html.slim

```diff
@@ -1,8 +1,19 @@
-h1= title 'Your Organizations'
+table
+  caption
+    h1= title 'Your Organizations'
+  thread
```

Comment:
This is a typo. Should be `thead`

Reply:
Oh my goodness. Fixed, committed, pushed! Thanks! @flipsasser